app/views/people/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @people = @people.first %>
```

Comment:
I'd recommend using a different name here instead of reassigning the value, maybe @person to signify that it's a singular instance